includes/managers/controls.php

```diff
@@ -1347,6 +1362,29 @@ private function promotion_switcher_control( $title, $id ): string {
 		</div>';
 	}
 
+	private function react_promotion_control( $title, $id ): string {
+		return '<div class="elementor-control-type-switcher elementor-label-inline e-control-motion-effects-promotion__wrapper">
```

Comment:
now this starts to look as a log of duplicated code. but to do some quick alignment of the infotip...
@hein-obox maybe, we should revert to the v1, when you tried to add an additional param to the promotion_control, to optionally render the "anchoring" element

Reply:
@mike-elementor Feel free to update this code to suite your requirements.

Reply:
@mike-elementor Let's try to avoid the duplication indeed.